README.md

```diff
@@ -1,2 +1,105 @@
-# ffc-pay-file-sender
-FFC Pay service to transfer files to DAX
+# FFC Pay file sender
+FFC Pay service to transfer files from payment service to Dynamics 365 (DAX)
+
+This function is triggered from a service bus message requesting a file transfer from Azure Blob Storage to an Azure File Share.
+
+The message contains the name of the file that should already have been written to a `dax` blob container, in a virtual directory named `outbound`.  The message should also include the target ledger for DAX. ie, `AP` or `AR`.
+
+If the file is present, the file will be copied to the DAX file share and the original blob will be moved to an archive folder.
+
+## Example message
+
+```
+{ 
+  "filename": "PFELM0002_AP_20220312231958 (SITI).csv",
+  "target": "AP"
+}
+```
+
+## Prerequisites
+
+- Node.js LTS 16+
+- access to an Azure blob storage account (see options below)
+- access to Azure file share storage account
+- [Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=v4%2Clinux%2Ccsharp%2Cportal%2Cbash)
```

Comment:
Should we give a specific version number?

Reply:
Yep.  Updated